File: sdk-dotnet/Examples/UserTasksExample/ItemRequestForm.cs
using LittleHorse.Sdk.UserTask;

namespace UserTasksExample;

public class ItemRequestForm
{
    [UserTaskField(
        DisplayName = "Your Request", 
        Description = "The item you are requesting.")]
    public string? RequestedItem;
    
    [UserTaskField(
        DisplayName = "Request Justification", 
        Description = "Why you need this request.")]
    public string? Justification;
}

File: examples/user-tasks/src/main/java/io/littlehorse/examples/ItemRequestForm.java
package io.littlehorse.examples;

import io.littlehorse.sdk.usertask.annotations.UserTaskField;

public class ItemRequestForm {

    @UserTaskField(
        description = "The item you are requesting.",
        displayName = "Your Request"
    )
    public String requestedItem;

    @UserTaskField(
        description = "Why you need this request.",
        displayName = "Request Justification"
    )
    public String justification;
}
